release-notes/3.0/preview/3.0.0-preview-known-issues.md

```diff
@@ -33,6 +33,23 @@ To update the path for every new terminal session, the `export` entry will need
 
 ## ASP.NET Core
 
+### Preview 9
+
+- **WebSocket connections to IIS In-Process servers will close after the client sends ~30MB**
+
+    WebSocket connections to IIS In-Process servers will close after the client sends ~30MB of data.
+
+    To workaround this issue you can host your app with the [OutOfProcess](https://docs.microsoft.com/aspnet/core/host-and-deploy/aspnet-core-module?view=aspnetcore-3.0#out-of-process-hosting-model) hosting model. Or you can change the `MaxRequestBodySize` limit in your app during configuration:
+  
+  ```csharp
+  var builder = new WebHostBuilder()
+      .UseIIS()
+      .ConfigureServices(services =>
+      {
+          services.Configure<IISServerOptions>(options => options.MaxRequestBodySize = long.MaxValue);
```

Comment:
How valid is this as a workaround? As IIS should still be enforcing the 30MB limit for non-websockets requests.